var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -65,12 +53,12 @@ class Coreneuron(CMakePackage):
     depends_on('flex@2.6:', type='build', when='+nmodl')
     depends_on('mpi', when='+mpi')
     depends_on('reportinglib', when='+report')
-    depends_on('libsonata-report@1.0:', when='@1.0.1:+report')
-    depends_on('libsonata-report@:0.1', when='@:1.0.0+report')
+    depends_on('libsonata-report@1.0:', when='@1.0.0.20210519:+report')
+    depends_on('libsonata-report@:0.1', when='@:1.0.0.20210518+report')
     depends_on('reportinglib+profile', when='+report+profile')
     depends_on('tau', when='+profile')
-    depends_on('caliper+mpi', when='@1.0.2a:+caliper+mpi')
-    depends_on('caliper~mpi', when='@1.0.2a:+caliper~mpi')
+    depends_on('caliper+mpi', when='@1.0.0.20210519:+caliper+mpi')
+    depends_on('caliper~mpi', when='@1.0.0.20210519:+caliper~mpi')
 
     # nmodl specific dependency
     depends_on('nmodl@0.3.0:', when='@1.0.0:+nmodl')
```

Comment:
I can't comment on the lines below this, but I realised that we have some references to old versions in these dependencies and below, e.g.
```python
@when('@0:0.16')
def get_cmake_args(self):
```
I think we should update these as we are dropping lots of versions.

Reply:
Agreed, check the new version! Thank you @olupton :)